pkg/controller/logstorage/esgateway.go

```diff
@@ -51,15 +52,18 @@ func (r *ReconcileLogStorage) createEsGateway(
 		r.status.SetDegraded("Error creating TLS certificate", err.Error())
 		return reconcile.Result{}, false, err
 	}
-	kibanaCertificate, err := certificateManager.GetCertificate(r.client, render.TigeraKibanaCertSecret, common.OperatorNamespace())
-	if err != nil {
-		reqLogger.Error(err, "failed to get Kibana tls certificate secret")
-		r.status.SetDegraded("Failed to get Kibana tls certificate secret", err.Error())
-		return reconcile.Result{}, false, err
-	} else if kibanaCertificate == nil {
-		reqLogger.Info("Waiting for internal Kibana tls certificate secret to be available")
-		r.status.SetDegraded("Waiting for internal Kibana tls certificate secret to be available", "")
-		return reconcile.Result{}, false, nil
+	var kibanaCertificate certificatemanagement.CertificateInterface
+	if !operatorv1.IsFIPSModeEnabled(install.FIPSMode) {
```

Comment:
Out of curiosity, why don't we care about the certificate in FIPS mode?

Reply:
Oh I guess we just don't install Kibana if FIPS mode is enabled, is that correct?

Reply:
That's right.